lib/releaselib.sh

```diff
@@ -1099,6 +1101,91 @@ release::docker::release () {
   return 0
 }
 
+# TODO(vdf): Consider collapsing this into release::docker::release and renaming
+#            that function AFTER the k8s.gcr.io Vanity Domain Flip (VDF).
+###############################################################################
+# Validates that image manifests have been pushed to a specified remote registry.
+# Uses 'skopeo inspect'.
+#
+# @param registry - docker registry
+# @param version - version tag
+# @param build_output - build output directory
+# @return 1 on failure
+release::docker::validate_remote_manifests () {
+  local registry=$1
+  local push_registry=$registry
+  local version=$2
+  local build_output=$3
+  local release_images=$build_output/release-images
```

Comment:
I think you should put variables you are assigning between `""`

Reply:
Done